doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -250,6 +251,27 @@ The following is an example of a record with non-trivial subtyping relation:
    E[Γ] ⊢ \mathsf{packType}@\{i\} =_{βδιζη}
    \mathsf{packType}@\{j\}~\mbox{ whenever }~i ≤ j
 
+Specifying cumulativity
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The variance of the universe parameters for a cumulative inductive may be specified by the user.
+
+For the following type, universe ``a`` has its variance automatically
+inferred (it is irrelevant), ``a'`` is required to be irrelevant,
+``b`` is covariant and ``c`` is invariant. With these annotations
+``b`` and ``c`` have less general variances than would be inferred.
```

Comment:
I suggest naming the universes `a b c d` and not `a a' b c`, just because one may think that `'` is a variance modifier.

Reply:
renamed